scaling.py

```diff
@@ -70,12 +70,11 @@ def wait_for_instance_launch(self, last_activity, count):
         @return: most recent activity
         """
         new_activities = []
-        looped = 0
+        TIMEOUT = 300
+        started = time.time()
         while len(new_activities) < count:
-            if looped > 6:
-                # TIMEOUT
+            if time.time() - started < TIMEOUT:
```

Comment:
I believe this should this be `> TIMEOUT` instead of `< TIMEOUT`.

I think this may be a carry over from `ecs.py`, where it was:
```
while time.time() - started < TIMEOUT: 
  # keep doing things
```